src/renderer/views/error/ErrorDiskSpaceView.tsx

```diff
@@ -4,29 +4,26 @@ import { Typography } from "@material-ui/core";
 import prettyBytes from "pretty-bytes";
 import { BLOCKCHAIN_STORE_PATH, REQUIRED_DISK_SPACE } from "../../../config";
 import * as Sentry from "@sentry/electron";
-import { useLocale } from "../../i18n";
-import { ErrorDiskSpace } from "../../../interfaces/i18n";
+import { T } from "@transifex/react";
 import { ipcRenderer } from "electron";
 
 const ErrorDiskSpaceView = () => {
   const classes = errorViewStyle();
 
-  const { locale } = useLocale<ErrorDiskSpace>("errorDiskSpace");
-
   useEffect(() => {
     ipcRenderer.send("mixpanel-track-event", "Launcher/ErrorDiskSpace");
     Sentry.captureException(new Error("Disk space is not enough."));
   }, []);
   return (
     <div className={classes.root}>
       <Typography variant="h1" gutterBottom className={classes.title}>
-        {locale("디스크 공간이 충분하지 않습니다")}
+        <T _str="Disk space is not enough." _tags="errorDiskSpace" />
       </Typography>
       <Typography>
-        {locale("필요한 여유 공간:")} {prettyBytes(REQUIRED_DISK_SPACE)}
+        <T _str="Required free space:" _tags="errorDiskSpace" /> {prettyBytes(REQUIRED_DISK_SPACE)}
```

Comment:
IMO, we can use like format string.

```suggestion
        <T _str="Required free space: {size}" size={prettyBytes(REQUIRED_DISK_SPACE)| _tags="errorDiskSpace" />
```

Reply:
Nice suggestion! Should we apply that to other string in this file as well?